src/components/Navigation.tsx

```diff
@@ -464,6 +466,23 @@ const Navigation: FC = () => {
                     },
                   )}
                 >
+                  <li className="p-side-navigation__item">
+                    <div className="p-side-navigation__link">
```

Comment:
```suggestion
                    <div className="p-side-navigation__link" title={loggedInUserName}>
```
We should add the title, in case the string is very long and gets truncated on hover we will then display the full content.